### DISCUSSION TITLE
Package 'nginx-plus-r24' has no installation candidate

### DISCUSSION THREAD

**Discussion by nergalex** [nginx/kubernetes-ingress#2035] [Q&A] [answered]
**Describe the bug**
During build, encountered error:
```
        "#8 37.74 E: Package 'nginx-plus-r24' has no installation candidate",
        "#8 37.74 E: Package 'nginx-plus-module-njs-r24' has no installation candidate",
```
See full Output:
[job_29855.txt](https://github.com/nginxinc/kubernetes-ingress/files/7274599/job_29855.txt)

**To Reproduce**
[create_nginx_ic_image.yaml.txt](https://github.com/nginxinc/kubernetes-ingress/files/7274577/create_nginx_ic_image.yaml.txt)

**Your environment**
* Version of the Ingress Controller - release version or a specific commit: 1.12.0 or 1.12.1
* docker==4.4.4
* Using NGINX Plus

**Additional context**
Last same task done with successful result: 8/19/2021
[job_29331.txt](https://github.com/nginxinc/kubernetes-ingress/files/7274590/job_29331.txt)
.

**Comment by jasonwilliams14** [comment, marked as answer]
@nergalex We identified the issue and made a new update to fix your issue. This should now be fully functional and working.
Make sure you clone the repo with the udated changes and then proceed to build your image per our documentation.

**Comment by nergalex** [comment]
Hi Jason,

Same issue (see output.json). However I can wait now for NAP 3.6 release if it is easier for you
[output.json.txt](https://github.com/nginxinc/kubernetes-ingress/files/7323568/output.json.txt)
repo cloned: git checkout v1.12.1

**Reply by lucacome** [reply]
Hi @nergalex,

it's fixed in https://github.com/nginxinc/kubernetes-ingress/releases/tag/v1.12.2, can you try that one?

**Comment by nergalex** [comment]
Hi @lucacome ,
Tried with 1.12.2, another error type was encountered: 
`nginx-plus-module-appprotect : Depends: app-protect-plugin (= 3.639.0-1~buster) but 3.671.0-1~buster is to be installed`

[output.json.txt](https://github.com/nginxinc/kubernetes-ingress/files/7330369/output.json.txt)

**Reply by jasonwilliams14** [reply]
@nergalex can you double check your build setup? I am unable to duplicate your problem in my lab.

the link that @lucacome provided works fine in my build environments. I can successfully build images across 1.12.x and 2.x as well.

**Comment by nergalex** [comment]
It's OK now. NGINX IC 2..0.2 built with NAP

Thank you guys
